### PR TITLE
3 commits: cleanup, readme updates, prepare for release

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: mono_repo/README.md [hunk 1]
```diff
@@ -27,4 +27,116 @@ Available commands:
 Run "mono_repo help <command>" for more information about a command.
 ```
 
+### Example
+
+In a repository with a single package in the `sub_pkg` directory, add a
+`mono_pkg.yaml` file:
```
Comment (jakemac53) [inline mono_repo/README.md]: `sub_pkg/mono_pkg.yaml` file

File: mono_repo/README.md [hunk 2]
```diff
@@ -27,4 +27,116 @@ Available commands:
...
+In a repository with a single package in the `sub_pkg` directory, add a
+`mono_pkg.yaml` file: 
+
+```yaml
+dart:
+ - dev
+
+stages:
+  - analyze:
+    - dartanalyzer
```
Comment (jakemac53) [inline mono_repo/README.md]: maybe show a group example also?

```
stages:
  - analyze:
    # Run tasks individually on their own job 
    - dartanalyzer
    - dartfmt
    # Or group similar tasks into one job 
    - group:
      - dartanalyzer
      - dartfmt
```

File: mono_repo/README.md [hunk 3]
```diff
@@ -27,4 +27,116 @@ Available commands:
...
+    - dartfmt
+  - unit_test:
+    - test
+```
+
+Running `mono_repo travis` in the root yields two files:
+
+#### `.travis.yml`
```
Comment (jakemac53) [inline mono_repo/README.md]: not sure showing the content provides a lot of value here? their contents should be transparent to the user?

and its just yet one more golden style test that has to be updated etc without providing much value

File: mono_repo/README.md (hunk 1 above)
Comment (kevmoo) [reply mono_repo/README.md]: ack

**Comment by kevmoo** [comment]
@jakemac53 – PTAL. We need a lot more docs, but I want to get v1 out soonish.

I'll add some issues to capture specifics we should cover.

**Pull request merged**